builtin/credential/ldap/backend.go

```diff
@@ -62,40 +58,6 @@ type backend struct {
 	*framework.Backend
 }
 
-func EscapeLDAPValue(input string) string {
```

Comment:
This function was moved with no change except to lowercase it so it's not exported.